execution-engine/src/hcall/buffer.rs

```diff
@@ -0,0 +1,138 @@
+//! The Veracruz host-call interface, and its implementation.
```

Comment:
Is this the correct header for the file?  I don't think this is the host-call interface, right?

Reply:
I have modified the rustdoc in this file: explicitly state glue code and will be removed in wasi change.